lib/viewcomponents/app.jsx

```diff
@@ -10,9 +10,9 @@ var App = React.createClass({
         return (
             <div id='app'>
                 <Graph model={this.props.model} />
-                <div id='model_editor'>
-                    <textarea onChange={this._onModelChange} value={JSON.stringify(this.props.model.getContents(), null, '  ')} />
-                </div>
+                {/*<div id='model_editor'>
+                    //<textarea onChange={this._onModelChange} value={JSON.stringify(this.props.model.getContents(), null, '  ')} />
+                //</div>*/}
```

Comment:
Because we're now using data model objects not pure JSON objects throughout the system, the model cannot be directly displayed or edited anymore.